src/main/cpp/subsystems/HatchIntake.cpp

```diff
@@ -22,11 +28,26 @@ std::shared_ptr<HatchIntake> HatchIntake::getInstance() {
   }
   return self;
 }
+static double calcFeedforward() {
```

Comment:
Put `calcFeedforward` inside the anonymous namespace.

Return a `constexpr` (look up how to do this).